orchestra/interface_api/project_management/views.py

```diff
@@ -34,7 +34,7 @@ def has_permission(self, request, view):
 
 class ProjectList(generics.ListCreateAPIView):
     permission_classes = (permissions.IsAuthenticated, IsProjectAdmin)
-    serializer_class = ProjectSerializer
+    serializer_class = ProjectSummarySerializer
```

Comment:
we only need the project ID/short_description/start_datetime to power autocomplete, so this brings our response size from a small number of MB to a small number of hundreds of KB

Reply:
cool!